source/connect/favorite-connections/import-export-cli/import.txt

```diff
@@ -12,4 +12,152 @@ Import Favorite Connections with the CLI
    :depth: 2
    :class: singlecol
 
-TODO: Write
+You can use the |compass-short| :abbr:`CLI (Command-Line Interface)` to
+import favorite connections. This lets you use saved connections from
+other workspaces or connections used by other team members.
+
+Prerequisite
+------------
+
+To import favorite connections, you must first :ref:`export the
+connections <compass-export-connections-cli>`.
+
+Import Unencrypted Favorite Connections
+---------------------------------------
+
+Use this procedure to import a list of favorite connections that is not
+encrypted with a passphrase.
+
+Procedure
+~~~~~~~~~
+
+To import unencrypted favorite connections with the |compass-short|
+:abbr:`CLI (Command-Line Interface)`, specify:
+
+- The path to the |compass| executable. The name and filepath of the
+  executable depend on your operating system.
+
+- The ``--import-connections`` flag with the value set to the
+  destination of the file containing favorite connections.
+
+Your operation should resemble the following prototype:
+
+.. code-block:: sh
+
+   <path-to-Compass-executable> --import-connections=<filename>
+
+Example
+~~~~~~~
+
+.. note::
+
+   The name and filepath of the executable depend on your operating
+   system. This example is for macOS.
```

Comment:
"The example in this section is..."

Also, perhaps not make a note? Slightly poor style to start a section with a note :)